@dimberman :
```
E       AssertionError: 'Erro[92 chars]ge to `dummyhostname.function`' != 'Erro[92 chars]ge to `dummyhostname.function`.This will be deprecated in 2.0.'
E       - Error: hostname_callable `dummyhostname:function` contains a colon instead of a dot. please change to `dummyhostname.function`
E       + Error: hostname_callable `dummyhostname:function` contains a colon instead of a dot. please change to `dummyhostname.function`.This will be deprecated in 2.0.
E       ?                                                                                                                               ++++++++++++++++++++++++++++++++

tests/upgrade/rules/test_hostname_callable_rule.py:29: AssertionError
```

--- a/airflow/utils/net.py
+++ b/airflow/utils/net.py
@@ -21,6 +21,7 @@ import importlib
 import socket
 from airflow.configuration import (conf, AirflowConfigException)
 
+
 def get_host_ip_address():
     return socket.gethostbyname(socket.getfqdn())
 
@@ -48,4 +49,4 @@ def get_hostname():
         callable = getattr(module, attr_name)
         return callable()
     else:
-        return conf.getimport('core', 'hostname_callable',fallback='socket.getfqdn')()
+        return conf.getimport('core', 'hostname_callable', fallback='socket.getfqdn')()

--- a/tests/upgrade/rules/test_hostname_callable_rule.py
+++ b/tests/upgrade/rules/test_hostname_callable_rule.py
@@ -27,8 +27,7 @@ class TestFernetEnabledRule(TestCase):
         self.assertEqual(
             result,
             "Error: hostname_callable `dummyhostname:function` "
-            "contains a colon instead of a dot. please change to `dummyhostname.function`."
-            "This will be deprecated in 2.0.",
+            "contains a colon instead of a dot. please change to `dummyhostname.function`",
         )
 
     @conf_vars({("core", "hostname_callable"): "dummyhostname.function"})

--- a/tests/utils/test_net.py
+++ b/tests/utils/test_net.py
@@ -19,6 +19,7 @@
 
 import unittest
 import mock
+import six
 
 from airflow.utils import net
 from tests.test_utils.config import conf_vars
@@ -75,7 +76,8 @@ class GetHostname(unittest.TestCase):
 
     @conf_vars({('core', 'hostname_callable'): 'tests.utils.test_net.missing_func'})
     def test_get_hostname_set_missing_2_0(self):
-        with self.assertRaisesRegex(
+        with six.assertRaisesRegex(
+            self,
             AirflowConfigException,
             re.escape(
                 'The object could not be loaded. Please check "hostname_callable" key in "core" section. '

